pollination/sky_irradiance/_raytracing.py

```diff
@@ -52,46 +48,24 @@ class SkyIrradianceRayTracing(DAG):
         spec={'type': 'string', 'enum': ['sensor', 'datetime']}
     )
 
-    @task(template=SplitGrid)
-    def split_grid(self, sensor_count=sensor_count, input_grid=sensor_grid):
-        return [
-            {'from': SplitGrid()._outputs.grids_list},
-            {'from': SplitGrid()._outputs.output_folder, 'to': '00_sub_grids'}
-        ]
-
-    # TODO: add a step to set divide_by to 1/timestep if  sky is cumulative.
-    @task(
-        template=DaylightCoefficient, needs=[split_grid],
-        loop=split_grid._outputs.grids_list, sub_folder='01_radiation',
-        sub_paths={'sensor_grid': '{{item.path}}'}
```

Comment:
We want to keep the `sub_paths` here. Otherwise, the recipe will copy all the sensor grids folder for each run.